fmriprep/workflows/epi.py

```diff
@@ -40,8 +40,10 @@
 def init_func_preproc_wf(bold_file, ignore, freesurfer,
                          bold2t1w_dof, reportlets_dir,
                          output_spaces, template, output_dir, omp_nthreads,
-                         fmap_bspline, fmap_demean, use_syn, force_syn,
-                         debug, output_grid_ref, layout=None):
+                         fmap_bspline, fmap_demean, debug, output_grid_ref,
+                         use_aroma, ignore_aroma_err,
+                         use_syn, force_syn, layout=None):
```

Comment:
Just to reduce the diff to a single addition, rather than 2-/4+.

```Python
fmap_bspline, fmap_demean, use_syn, force_syn,
use_aroma, ignore_aroma_err,
debug, output_grid_ref, layout=None):
```